src/main/java/com/teamwizardry/librarianlib/features/particlesystem/modules/VelocityUpdateModule.kt

```diff
@@ -32,6 +32,7 @@ class VelocityUpdateModule(
         position.load(particle)
         if(previousPosition != null) {
             position.contents.copyInto(previousPosition.contents)
+            previousPosition.store(particle)
```

Comment:
This seems unrelated to current PR, it's a fix I assume?

Reply:
Yeah